### ISSUE TITLE
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate   "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)

### ISSUE THREAD

**Issue by sunyclj** [cszn/BSRGAN#28]
python main_train_psnr.py --opt options/train_bsrgan_x4_psnr.json 运行至model.update_learning_rate(current_step)报上面错误；pytorch版本为1.7.1
